lib/srv/usermgmt.go

```diff
@@ -133,7 +133,11 @@ type HostUserManagement struct {
 
 var _ HostUsers = &HostUserManagement{}
 
-var sudoersSanitizationMatcher = regexp.MustCompile(`\.|~`)
+// Under the section "Including other files from within sudoers":
+//           https://man7.org/linux/man-pages/man5/sudoers.5.html
+// ., ~ will cause a file not to be read and these can be included in
```

Comment:
Minor nitpick here. Could be just me but I was confused a little about which characters are actually causing the problem. To me,`., ~` feels like `.`, `,`, ` `, and `~`.